aci/resource_aci_rtctrlattrp.go

```diff
@@ -359,7 +359,7 @@ func getRemoteRtctrlSetRedistMultipath(client *client.Client, dn string) (*model
 	}
 	rtctrlSetRedistMultipath := models.RedistributeMultipathActionFromContainer(rtctrlSetRedistMultipathCont)
 	if rtctrlSetRedistMultipath.DistinguishedName == "" {
-		return nil, fmt.Errorf("rtctrl Set Redist Multipath %s not found", dn)
+		return nil, fmt.Errorf("Route Control Set Redist Multipath %s not found", dn)
```

Comment:
Should we change Redist into Redistribute?

Reply:
Sure